package.json

```diff
@@ -5,32 +5,43 @@
   "main": "build/index.js",
   "jsnext:main": "src/index.js",
   "scripts": {
-    "test": "eslint src && flow",
+    "test": "jest",
+    "lint": "eslint src",
```

Comment:
We also want to lint the tests. `"lint": "eslint ."` would be the easiest and then we can [ignore](https://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories) whatever shouldn't be linted.

Reply:
Referring to he comment above ^ 

What is the correct way to remove something from github that was added to .gitignore file after it was pushed up? ie. (recipes folder)